src/bloom_filter/bloom_test.rs

```diff
@@ -1,56 +1,97 @@
 #[cfg(test)]
-mod tests {
-    use super::super::BloomFilter;
```

Comment:
Using `mod tests` seems to be the Rust way of doing unit tests (https://doc.rust-lang.org/rust-by-example/testing/unit_testing.html). Is there a reason you are changing this? With your changes I am not sure that all the functions are covered by `#[cfg(test)]`

Reply:
I think that was part of a refactor into separate file. 
Unit tests are not required to be in the same file to be valid, but I forgot about those specific changes.
I think it was since refactoring out into a file makes mod tests and the import redundant, but I'll have to check.